src/react/deckgl.js

```diff
@@ -50,7 +51,8 @@ const defaultProps = {
   effects: [],
   onWebGLInitialized: noop,
   onLayerClick: noop,
-  onLayerHover: noop
+  onLayerHover: noop,
+  onLayerDrawn: noop
```

Comment:
Is the name of this callback (i.e. `onLayerDrawn`) appropriate? It aligns with the existing callbacks, but this is not called for a specific layer right? It is more like `onAfterRender`, `onAllLayersDrawn`, etc.

Also, since this is the public API, it should probably be described in the documentation.